openstack/gaussdbnosql/v3/ListFlavors.go

```diff
@@ -0,0 +1,67 @@
+package v3
+
+import (
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/extract"
+)
+
+type ListFlavorsOpts struct {
+	// Region where the instance is deployed.
+	// Valid value:
+	// The value cannot be empty. For details about how to obtain this parameter value, see Regions and Endpoints.
+	Region string `q:"region"`
+	// engine_name	No	Database type
+	// 	If the value is cassandra, the GaussDB(for Cassandra) DB instance specifications are queried,
+	// 	If this parameter is not transferred, the default value is cassandra.
+	EngineName string `q:"engine_name,omitempty"`
+}
+
+func ListFlavors(client *golangsdk.ServiceClient, opts ListFlavorsOpts) (*ListFlavorsResponse, error) {
+	q, err := golangsdk.BuildQueryString(opts)
+	if err != nil {
+		return nil, err
+	}
+
+	// GET https://{Endpoint}/v3/{project_id}/flavors
+	raw, err := client.Get(client.ServiceURL("flavors")+q.String(), nil, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	var res ListFlavorsResponse
+	err = extract.Into(raw.Body, &res)
+	return &res, err
+}
+
+type ListFlavorsResponse struct {
+	// Total number of records
+	TotalCount int32 `json:"total_count"`
+	// Instance specifications.
+	Flavors []ListFlavorsResult `json:"flavors"`
+}
+
+type ListFlavorsResult struct {
+	// Engine name
+	EngineName string `json:"engine_name"`
+	// Engine version
+	EngineVersion string `json:"engine_version"`
+	// Number of vCPUs
+	Vcpus string `json:"vcpus"`
+	// Memory size in megabytes (MB)
+	Ram string `json:"ram"`
+	// Resource specification code
+	// Example: geminidb.cassandra.8xlarge.4
+	// NOTE
+	// 	geminidb.cassandra indicates GaussDB(for Cassandra).
```

Comment:
this strange ? symbols, can you remove it? it can cause some lint issues in future